ts/features/itwallet/machine/eid/actors.ts

```diff
@@ -1,47 +1,49 @@
-import { openAuthenticationSession } from "@pagopa/io-react-native-login-utils";
 import { fromPromise } from "xstate5";
-import { LocalIdpsFallback } from "../../../../utils/idps";
-import { getIdpLoginUri } from "../../../../utils/login";
+import * as O from "fp-ts/lib/Option";
+import * as issuanceUtils from "../../common/utils/itwIssuanceUtils";
 import { StoredCredential } from "../../common/utils/itwTypesUtils";
+import { assert } from "../../../../utils/assert";
+import { ensureIntegrityServiceIsReady } from "../../common/utils/itwIntegrityUtils";
 import {
-  getAttestation,
   getIntegrityHardwareKeyTag,
   registerWalletInstance
 } from "../../common/utils/itwAttestationUtils";
+import { useIOStore } from "../../../../store/hooks";
+import { itwIntegrityKeyTagSelector } from "../../issuance/store/selectors";
+import { type Identification } from "./context";
 
-export const createEidIssuanceActorsImplementation = () => ({
+export type RequestEidActorParams = {
+  integrityKeyTag: string | undefined;
+  identification: Identification | undefined;
+};
+
+export const createEidIssuanceActorsImplementation = (
+  store: ReturnType<typeof useIOStore>
+) => ({
   createWalletInstance: fromPromise<string>(async () => {
-    try {
-      const hardwareKeyTag = await getIntegrityHardwareKeyTag();
-      await registerWalletInstance(hardwareKeyTag);
-      return Promise.resolve(hardwareKeyTag);
-    } catch (e) {
-      return Promise.reject(e);
-    }
-  }),
+    const storedIntegrityKeyTag = itwIntegrityKeyTagSelector(store.getState());
```

Comment:
Actors must be pure functions, what about saving this information in the context when the machine is being initialized? So we could pass the `storedIntegrityKeyTag` as input to this actor.